airflow-core/tests/unit/models/test_team.py

```diff
@@ -0,0 +1,70 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import uuid
+from unittest.mock import MagicMock, patch
+
+from airflow.models.team import Team
+
+
+class TestTeam:
+    """Unit tests for Team model class methods."""
+
+    def test_get_all_teams_id_to_name_mapping_with_teams(self):
```

Comment:
Since there is a DB query in the src function for this test, I was wondering if it's worth making this a DB test?

Reply:
I explicitly chose not to since it is a very simple query and if I can avoid it it's nice to be able to run the tests without a DB instance. But if you feel strongly I can make a real DB query here

Reply:
I see, I'm good either way so feel free to resolve